contentcuration/contentcuration/models.py

```diff
@@ -650,6 +654,30 @@ def get_channel_thumbnail(channel):
 )
 
 
+class PermissionCTE(With):
+    def __init__(self, model, user_id, **kwargs):
+        tree_id_fields = [
```

Comment:
We can probably create this array in the module scope, as it doesn't change per query.